packages/kbn-i18n/src/core/error_handler.ts

```diff
@@ -6,6 +6,11 @@
  * Side Public License, v 1.
  */
 
-declare module 'intl-relativeformat' {
-  export let defaultLocale: string;
-}
+import { OnErrorFn, IntlErrorCode } from '@formatjs/intl';
+
+export const handleIntlError: OnErrorFn = (error) => {
+  // Dont through on missing translations.
```

Comment:
Typo:
> Don't throw on missing translations

Shouldn't we log an error at least?

Reply:
No i dont think we should log an error for missing translations.
1. It is not the responsibilty of the developers or anyone looking at the logs to see those since this would only be useful for translators.
2. We'll almost always have missing translations since we do translations in cycles so it'll be just noise everywhere.